cms/djangoapps/contentstore/views/item.py

```diff
@@ -457,6 +459,10 @@ def _save_xblock(user, xblock, data=None, children_strings=None, metadata=None,
         else:
             data = old_content['data'] if 'data' in old_content else None
 
+        if fields:
```

Comment:
All the other arguments have special handling besides simple assignment, so I think they can't be consolidated into `fields` the same way the others do.